src/service/InteractiveExposeV2Service/InteractiveExposeV2Controller.ts

```diff
@@ -0,0 +1,18 @@
+import { APIClient, APIMapping } from '../../http';
+import { InteractiveExposeV2ServiceTypes } from './InteractiveExposeV2Service.Types';
+
+export class InteractiveExposeV2Controller extends APIClient {
+    constructor() {
+        super(APIMapping.interactiveExposeV2LambdaService);
+    }
+    /**
+     *
+     * sends an exposé
+     *
+     */
+    iexBlast(templateId: string, payload: InteractiveExposeV2ServiceTypes.BlastRequest) {
```

Comment:
`sendBlast` is a better name, because `iex` is not needed as prefix, because you are already in the "iex" context here

```suggestion
    sendBlast(templateId: string, payload: InteractiveExposeV2ServiceTypes.BlastRequest) {
```

Reply:
Agree